bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -64,12 +64,12 @@ async def oldest_command(self, ctx: Context, update_cache: bool = True) -> None:
 
     @nomination_group.command(name='watch', aliases=('w', 'add', 'a'), root_aliases=("nominate",))
     @has_any_role(*STAFF_ROLES)
-    async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: str) -> None:
+    async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: Optional[str] = '') -> None:
```

Comment:
The `Optional` annotation still needs to be removed here, and left like it was before.
```suggestion
    async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: str = '') -> None:
```